services/importer/spec/acceptance/gpx_spec.rb

```diff
@@ -28,11 +40,14 @@
   it 'imports GPX files' do
     filepath    = path_to('route2.gpx')
     downloader  = CartoDB::Importer2::Downloader.new(@user.id, filepath)
+    unpacker    = CartoDB::Importer2::Unp.new(nil, ogr2ogr2_unpack_options)
     runner      = CartoDB::Importer2::Runner.new(pg: @user.db_service.db_configuration_for,
                                                  downloader: downloader,
                                                  log: CartoDB::Importer2::Doubles::Log.new(@user),
-                                                 user: @user
+                                                 user: @user,
+                                                 unpacker: unpacker
```

Comment:
How come we're passing in a unpacker now? What was used before? Is this a `CartoDB::Importer2::Runner` interface change?

Reply:
The unit tests fail because the unpacker is using the wrong ogr2ogr (in Bloomberg environment).  Unfortunately the only way to send this information is by manually creating the unpacker and passing it in.  The unpacker should more or less match the one created by default in https://github.com/cartodb-org/cartodb/blob/68c33299886bf2ccebee6248fa14d75193ece81e/services/importer/lib/importer/runner.rb#L54

Reply:
Very nice ✨